dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImplTest.java

```diff
@@ -443,8 +444,8 @@ public void optimize () throws Exception {
 
         //Build the index names
         String timeStamp = String.valueOf( new Date().getTime() );
-        String workingIndex = ContentletIndexAPIImpl.ES_WORKING_INDEX_NAME + "_" + timeStamp;
-        String liveIndex = ContentletIndexAPIImpl.ES_LIVE_INDEX_NAME + "_" + timeStamp;
+        String workingIndex = IndexType.WORKING.getPrefix() + "_" + timeStamp;
+        String liveIndex = IndexType.LIVE.getPrefix() + "_" + timeStamp;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'liveIndex' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)